src/js/common/components/authentication/VerifyPrompt.jsx

```diff
@@ -1,31 +1,40 @@
+import PropTypes from 'prop-types';
 import React from 'react';
-import { handleVerify } from '../../helpers/login-helpers.js';
 
 class VerifyPrompt extends React.Component {
-
   constructor(props) {
     super(props);
-    this.handleVerify = handleVerify;
+    this.handleVerify = this.handleVerify.bind(this);
   }
 
   componentDidMount() {
-		// event for google analytics that we prompted to verify account
     window.dataLayer.push({ event: 'verify-prompt-displayed' });
   }
 
+  handleVerify() {
+    window.dataLayer.push({ event: 'verify-link-clicked' });
+    const myVerifyUrl = this.props.verifyUrl;
+    if (myVerifyUrl) {
+      window.dataLayer.push({ event: 'verify-link-opened' });
+      const receiver = window.open(`${myVerifyUrl}&op=signin`, '_blank', 'resizable=yes,scrollbars=1,top=50,left=500,width=500,height=750');
+      receiver.focus();
+    }
+  }
+
   render() {
     return (
       <div className="row primary">
         <div className="medium-12 small-12 columns">
           <div className="react-container">
             <h1>Verify your Identity with ID.me</h1>
             <p>You need to verify your identity to access this part of Vets.gov.</p>
-            <p>To access Vets.gov services, you'll need to verify your identity through ID.me, a third party service that provides strong identity verification. We have added this protection to increase security for your information. Here's what you'll need:
+            <p></p>
+            <div>To access Vets.gov services, you'll need to verify your identity through ID.me, a third party service that provides strong identity verification. We have added this protection to increase security for your information. Here's what you'll need:
```

Comment:
I don't remember where I saw this warning, so reverted for now. Will see if it returns.